WolvenKit.App/Helpers/ArchiveXlHelper.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Splat;
```

Comment:
What is Splat doing here? I would prefer to avoid adding more libraries than necessary--even though it's used in the main csproj

Reply:
it's required to get the service